server/models/users.py

```diff
@@ -48,10 +57,6 @@ def find_by_id(cls, user_id):
     def verify_password(self, password):
         return flask_bcrypt.check_password_hash(self.password, password)
 
-    @classmethod
-    def find_by_id(cls, user_id):
-        return cls.query.filter_by(id=user_id).first()
-
```

Comment:
Why did we get rid of this? From my understanding, we're going to be storing the user id in our jwt tokens/session - if that's the case, I'll need to be able to look up the current user's information in order to be able to check/utilize a user's google auth credentials (that's also what I'm using to store their credentials.)

Reply:
It was a duplicate! I just deleted one of them